blender/flake.nix

```diff
@@ -47,17 +47,18 @@
         };
 
       mkTest = { blender }:
-        pkgs.runCommand "blender-test" { buildInputs = [ blender ]; }
+        new-pkgs.runCommand "blender-test" { buildInputs = [ blender ]; }
           ''
             blender --version
             touch $out
           '';
 
     in {
 
-      overlays.default = final: prev: {
+      overlays.default = pkgs: pkgs-super: {
```

Comment:
I think `nix flake check` requires these to be named `final` and `prev`.

Reply:
Wow, that’s weird. Thanks.

Reply:
I’ve updated it.